### DISCUSSION TITLE
Need advice regarding setting up Rocket.Chat locally

### DISCUSSION THREAD

**Discussion by vivekchavan14** [RocketChat/Rocket.Chat#31825] [General]
I'm trying to setup RC on ubuntu, i have gone through the developers guide, i have installed corepack, meteor and also iam using node 14, 
i'm able to run "yarn" command but when i do "yarn dsv" i get an error "Cant start Mongo Server", can anyone help me on how can i fix this?

**Comment by rezerov** [comment]
Are you running `yarn build` before `yarn dsv`?

If yes, try running `yarn add deasync`, and then build and run the development server.

**Reply by vivekchavan14** [reply]
I have resolved the issue, i'm able to set it up locally now, Thanks!

**Reply by usersdfg** [reply]
Can you give me commands steps, which you run to start it locally, I am try from long time ,please share the commands
